model-engine/model_engine_server/api/app.py

```diff
@@ -18,10 +22,16 @@
 from model_engine_server.api.model_endpoints_v1 import model_endpoint_router_v1
 from model_engine_server.api.tasks_v1 import inference_task_router_v1
 from model_engine_server.api.triggers_v1 import trigger_router_v1
-from model_engine_server.common.datadog_utils import get_request_id
-from model_engine_server.core.loggers import filename_wo_ext, make_logger
-from starlette.middleware.base import BaseHTTPMiddleware
+from model_engine_server.core.loggers import (
+    filename_wo_ext,
+    get_request_id,
+    make_logger,
+    set_request_id,
+)
+
+logger = make_logger(filename_wo_ext(__name__))
 
+patch(fastapi=True)
```

Comment:
Is this manual patching still necessary? I think you were adding this while testing, but hopefully this shouldn't be necessary.

Reply:
just tested 1 more time and confirmed patching is not necessary